applications/GeoMechanicsApplication/tests/cpp_tests/test_equation_of_motion.cpp

```diff
@@ -67,7 +67,11 @@ KRATOS_TEST_CASE_IN_SUITE(CalculateMassMatrix2D6NDiffOrderGivesCorrectResults, K
     KRATOS_CHECK_MATRIX_NEAR(mass_matrix, expected_mass_matrix, 1e-4)
 }
 
-KRATOS_TEST_CASE_IN_SUITE(CalculateMassMatrix3D4NGivesCorrectResults, KratosGeoMechanicsFastSuite)
+class StubConstitutiveLaw : public ConstitutiveLaw
+{
+};
```

Comment:
As far as I can see, this class is not being used. If that is correct, please remove it.

Reply:
You're right, removed!